{"uuid": "afa136f9-417a-469e-a17a-c1dd1d3ebf05", "children": ["efc76134-186f-4d18-bc3b-407d575061b6"], "befores": [{"name": "_xunit_setup_method_fixture_TestLogin", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectionError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))\n", "trace": "  File \"D:\\python\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"D:\\python\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\python.py\", line 821, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\python\\lib\\site-packages\\_pytest\\python.py\", line 717, in _call_with_optional_argument\n    func()\n  File \"D:\\apptest\\script\\login\\test_login.py\", line 43, in setup_method\n    self.driver = UtilsDriver.get_mp_driver()\n  File \"D:\\apptest\\utils.py\", line 37, in get_mp_driver\n    cls._mp_session = cls._mp_driver.session(cls.APP_CONFIG['bundle_id'])\n  File \"D:\\python\\lib\\site-packages\\wda\\__init__.py\", line 591, in session\n    res = self.http.post('session', payload)\n  File \"D:\\python\\lib\\site-packages\\wda\\__init__.py\", line 409, in _fetch\n    response = httpdo(url, method, data)\n  File \"D:\\python\\lib\\site-packages\\wda\\__init__.py\", line 125, in httpdo\n    return _unsafe_httpdo(url, method, data)\n  File \"D:\\python\\lib\\site-packages\\wda\\__init__.py\", line 162, in _unsafe_httpdo\n    response = request_session.request(method,\n  File \"D:\\python\\lib\\site-packages\\requests\\sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n  File \"D:\\python\\lib\\site-packages\\requests\\sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n  File \"D:\\python\\lib\\site-packages\\requests\\adapters.py\", line 682, in send\n    raise ConnectionError(err, request=request)\n"}, "start": 1736404565743, "stop": 1736404567930}], "start": 1736404565743, "stop": 1736404568187}